src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -48,5 +48,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[15.0,16.0)" DisplayName="Windows PDB reader/writer" />
```

Comment:
> No, the compilers do not depend on this Willow VSIX.

Are we sure?  This is the compiler extension which installs into Visual Studio when we F5.  That isn't how we traditionally ship.  I'm struggling to see how the Roslyn VSIX would need a dependency here but not that one.

It's a bit of a mute point because we don't ship that.  It's more for our own dogfooding.  More curious than anything about this.  Not blocking feedback.